sdk/synapse/azure-analytics-synapse-accesscontrol/src/main/java/com/azure/analytics/synapse/accesscontrol/models/AccessControlsGetRoleAssignmentsHeaders.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
The generated models should belong to `.implementation` package?

Reply:
We want to expose them to developers. See words of @JonathanGiles at below
> RoleAssignmentDetails is a class in the implementation package, which means it cannot be exposed through public API. You need to move RoleAssignmentDetails out of the implementation package if you intend for it to be used by developers.